ArduPlane/quadplane.h

```diff
@@ -115,8 +115,11 @@ class QuadPlane
     // return true when tailsitter frame configured
     bool is_tailsitter(void) const;
 
-    // return true when flying a control surface only tailsitter tailsitter
-    bool is_contol_surface_tailsitter(void) const;
+    // return true when flying a control surface only tailsitter
+    bool is_control_surface_tailsitter(void) const;
+
+    // true when flying a tilt-vectored tailsitter
+    bool _is_vectored;
```

Comment:
as mentioned on the dev call, there are two variables named, "_is_vectored".  one here in the vechicle code and another in the AC_ATtitude controller.